c7n/resources/aws.py

```diff
@@ -781,9 +781,14 @@ def initialize_policies(self, policy_collection, options):
             available_regions = service_region_map.get(
                 resource_service_map.get(resource_type), ())
 
-            # its a global service/endpoint, use user provided region
-            # or us-east-1.
-            if not available_regions and options.regions:
+            if resource_service_map.get(resource_type) == 'route53domains':
```

Comment:
we should just set global = False for this resource

Reply:
Done